cli/remotebazel/remotebazel.go

```diff
@@ -825,9 +826,20 @@ func HandleRemoteBazel(args []string) (int, error) {
 		runner = "grpcs://" + runner
 	}
 
+	apiKey := arg.Get(bazelArgs, "remote_header=x-buildbuddy-api-key")
+	if apiKey == "" {
+		apiKey, err = storage.ReadRepoConfig("api-key")
+		if err != nil {
+			return 1, status.WrapError(err, "read api key from bb config")
+		}
+	}
+	if apiKey == "" {
+		return 1, status.UnauthenticatedError("missing API key - run `bb login` to authorize")
```

Comment:
Thoughts on just running `bb login` for them in this case?

Reply:
I like that - updated